doc/UsersGuide/BuildingRunningTesting/tutorials/tutorial4.inc

```diff
@@ -0,0 +1,550 @@
+.. _fcst4:
+
+Sample Forecast #4: Halloween Storm
+=======================================
+
+**Objectives:** 
+   * Compare forecast outputs for similar experiments that use different :term:`IC/LBC <ics/lbcs>` sources. 
+   * Use verification tools to assess forecast quality. 
+
+Weather Summary
+--------------------
+
+A line of severe storms brought strong winds, flash flooding, and tornadoes to the eastern half of the US.
+
+**Weather Phenomena:** Flooding and high winds
+
+   * `Storm Prediction Center (SPC) Storm Report for 20191031 <https://www.spc.noaa.gov/climo/reports/191031_rpts.html>`_
+
+.. figure:: https://github.com/ufs-community/ufs-srweather-app/wiki/Tutorial/HalloweenStorm.gif
+   :alt: Radar animation of the Halloween Storm that swept across the Eastern United States in 2019. 
+
+   *Halloween Storm 2019*
+
+Data
+-------
+
+Data for the Halloween Storm is publicly available in S3 data buckets. The Rapid Refresh (`RAP <https://rapidrefresh.noaa.gov/>`_) data can be downloaded from the `SRW App data bucket <https://registry.opendata.aws/noaa-ufs-shortrangeweather/>`_ using ``wget``. Make sure to issue the command from the folder where you want to place the data.
+
+.. code-block:: console
+
+   wget https://noaa-ufs-srw-pds.s3.amazonaws.com/develop-20240618/halloween_rap.tgz
```

Comment:
The link is provided here is dead.  The only data I can find for the Halloween storm is `HalloweenStormData.tar.gz`, but it looks like this data is not the same as what is included in `halloween_rap.tgz`.  It is unclear where the `halloween_rap.tgz` data can be found following the update to the S3 bucket.